classes/publication/PublicationCategory.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace PKP\publication;
+
+use Illuminate\Database\Eloquent\Model;
+
+class PublicationCategory extends Model
+{
+    protected $table = 'publication_categories';
+    protected $primaryKey = 'publication_category_id';
+    public $timestamps = false;
+
+
+    protected $fillable = [
+        'publication_id', 'category_id'
+    ];
+
+    /**
+     * Get the categories associated with a publication.
+     *
+     */
+    public static function getCategoriesByPublicationId(int $publicationId)
```

Comment:
Use local [scope functions](https://laravel.com/docs/10.x/eloquent#local-scopes), allowing chaining of constraints. [Example](https://github.com/pkp/pkp-lib/blob/main/classes/stageAssignment/StageAssignment.php#L100)